doc/RFQ.md

```diff
@@ -1,18 +1,26 @@
 # RFQ
 
-RFQ (Request For Quote) is a contract that settles a trade between two parties. Normally it requires an off-chain quoting system which provides quotes and signatures from certain makers. A user may request for a quote of a specific trading pair and size. Upon receiving the request, any maker willing to trade can respond with a quote to the user. If the user accepts it, then both parties will sign the trading order and submit the order with signatures to the RFQ contract. After verifying signatures, the RFQ contract will transfer tokens between user and maker accordingly.
+The RFQ (Request For Quote) contract facilitates the settlement of trades between two parties: a market maker and an user. We provide an off-chain quoting system that handles the quoting process. Users can request quotes for specific trading pairs and sizes. Upon receiving a request, any interested maker can provide a quote to the user. If the user accepts the quote, both parties will sign the trading order and submit it ot the RFQ contract along with their signatures. The RFQ contract verifies the signatures and executes the token transfers between the user and the market maker.
 
 ## Order option flags
 
-There are two options that the maker of a RFQ offer can set. The option flags is a uint256 field in the offer.
+The maker of an RFQ offer can specify certain options using a `uint256` field in the offer, referred to as option flags:
 
--   Partial fill : Whether the Offer can be filled partially or not (but once).
--   Contract call : Whether the Offer can be filled by a contract or not.
+-   `FLG_ALLOW_CONTRACT_SENDER` : Determines whether the RFQ offer can be filled by a contract.
+-   `FLG_ALLOW_PARTIAL_FILL` : Whether a RFQ offer can be filled partially or not (but once).
```

Comment:
What does `(but once)` mean here?

Reply:
it indicates an RFQ order can be partial filled, but can only be filled once, regardless of any remaining quantity.

Reply:
Revised version:  Determines whether an RFQ offer can be partially filled. However, each RFQ order can only be filled once, regardless of whether it's fully or partially filled.